data/soapvalidator/MailClient/Search/DelayedIndexing/ZCS-8515-8517.xml

```diff
@@ -0,0 +1,608 @@
+<t:tests xmlns:t="urn:zimbraTestHarness">
+
+<t:property name="test_account1.name" value="test1${TIME}${COUNTER}@${defaultdomain.name}"/>
+<t:property name="test_account2.name" value="test2${TIME}${COUNTER}@${defaultdomain.name}"/>
+<t:property name="test_account3.name" value="test3${TIME}${COUNTER}@${defaultdomain.name}"/>
+<t:property name="test_account4.name" value="test4${TIME}${COUNTER}@${defaultdomain.name}"/>
+<t:property name="mail_content" value="Content in the message is content${TIME}${COUNTER}"/>
+
+<t:property name="server.zimbraAdmin" value="${zimbraServer.name}"/>
+
+<t:test_case testcaseid="Ping" type="always">
+    <t:objective>basic system check</t:objective>
+
+    <t:test required="true">
```

Comment:
required=''true'' should be removed. This blocks tests where unauth ping is not enabled.